<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>993</width>
    <height>566</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>291</width>
      <height>261</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="inputBlock">
     <item>
      <layout class="QVBoxLayout" name="labels">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Project title</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Task</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Performer</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Supervisor</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Issue date</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Due date</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Deadline</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLineEdit" name="editTitle"/>
       </item>
       <item>
        <widget class="QLineEdit" name="editTask"/>
       </item>
       <item>
        <widget class="QLineEdit" name="editPerformer"/>
       </item>
       <item>
        <widget class="QLineEdit" name="editSupervisor"/>
       </item>
       <item>
        <widget class="QDateEdit" name="editIssueDate"/>
       </item>
       <item>
        <widget class="QDateEdit" name="editDueDate"/>
       </item>
       <item>
        <widget class="QDateEdit" name="editDeadline"/>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QTextEdit" name="output">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>10</y>
      <width>671</width>
      <height>531</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>280</y>
      <width>291</width>
      <height>261</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QPushButton" name="openFileButton">
       <property name="text">
        <string>Open file</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="saveFileButton">
       <property name="text">
        <string>Save file</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="addItemButton">
       <property name="text">
        <string>Add item</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="supervisorProjectsButton">
       <property name="text">
        <string>Supervisor projects</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="performerTasksButton">
       <property name="text">
        <string>Performer tasks</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="projectPerformersButton">
       <property name="text">
        <string>Project performers</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="missedDeadlinePerformersButton">
       <property name="text">
        <string>Out of deadline performers</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
